src/plugins/condition/utils/evaluator.js

```diff
@@ -1,7 +1,31 @@
-export const computeConditionForAny = (args) => {
-    return false;
+/**
+ * Returns true only if at least one of the results is true
+ **/
+export const computeConditionForAny = (resultMap) => {
+    let result = false;
+    for (let key in resultMap) {
+        if (resultMap.hasOwnProperty(key)) {
+            result = resultMap[key];
+            if (result) {
+                break;
+            }
+        }
+    }
+    return result;
 };
 
```

Comment:
Why not refactor these as computeCondition(trigger, resultMap)?

Reply:
Mostly for clarity.

Reply:
Done